### PR TITLE
Completed typescript-and-json - Nader Motamedi

### PR DESCRIPTION
**Pull request by nmotamedi**
Screen Shot 2024-02-16 at 10 07 32 AM

### CODE REVIEW

File: typescript-and-json/main.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+];
+
+console.log('Array of books: ', array);
+console.log('type of array: ', typeof array);
+
+const booksJSON = JSON.stringify(array);
+console.log('booksJSON: ', booksJSON);
+console.log('type of booksJSON: ', typeof booksJSON);
+
+const studentJSON = '[{"Number id":2302, "String name":"Nader Motamedi"}]';
```
Comment (shawnkost) [inline typescript-and-json/main.ts:18]: Couple things!

1. No array, just an object
1. Properties shouldn't have spaces in them. The property names should just be `id` & `name` respectively

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple things here Nader!

File: typescript-and-json/main.ts (hunk 1 above)
Comment (nmotamedi) [reply typescript-and-json/main.ts:18]: I see!

**Comment by nmotamedi** [comment]
Screen Shot 2024-02-16 at 10 15 51 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**